packages/monitors/src/OptimisticOracleContractMonitor.js

```diff
@@ -210,12 +210,10 @@ class OptimisticOracleContractMonitor {
     this.lastSettlementBlockNumber = this._getLastSeenBlockNumber(latestEvents);
   }
 
-  // Returns helper method for converting collateral token associated with financial contract to human readable form.
-  async _getCollateralDecimalsConverted(financialContractAddress) {
-    const financialContract = new this.web3.eth.Contract(getAbi("FeePayer"), financialContractAddress);
```

Comment:
Because we currently don't set the `currency` in the `event` object coming from the `OptimisticOracleEventClient`, we infer the currency address by assuming that the `requester` is a financial contract that we can call `collateralCurrency` on. This is a bad assumption so this change just seems like a generally good feature to have.

Reply:
Agreed, this is way better.